R/archive.R

```diff
@@ -453,8 +454,12 @@ as_epi_archive <- function(
     additional_metadata = NULL,
     compactify = NULL,
     clobberable_versions_start = NULL,
-    versions_end = NULL) {
+    versions_end = NULL, ...) {
   assert_data_frame(x)
+  x <- rename(x, ...)
```

Comment:
question: do we need to do a `tryCatch` here, too?

Reply:
The error in this case is
```
as_epi_archive(rename(dt, weirdName = version), version = weirdName, version = time_value)
Error in `rename()` at ]8;line = 461:col = 3;file:///home/dsweber/allHail/delphi/epiprocess/R/archive.Repiprocess/R/archive.R:461:3]8;;:
! Names must be unique.
✖ These names are duplicated:
  * "version" at locations 1 and 2.
Run `]8;;rstudio:run:rlang::last_trace()rlang::last_trace()]8;;` to see where the error occurred.
```
which I think is more obvious what went wrong. For the other rename, it's buried a little deeper why the names are redundant, so I wanted to give some context